src/main/resources/application.yml

```diff
@@ -20,22 +20,22 @@ extraction:
       caseType: "GrantOfRepresentation"
       type: "jsonlines"
       prefix: "CCD-PROB-GOR"
-    - container: "caveat"
-      caseType: "Caveat"
-      type: "jsonlines"
-      prefix: "CCD-PROB-CAVEAT"
-    - container: "willlodgement"
-      caseType: "WillLodgement"
-      type: "jsonlines"
-      prefix: "CCD-PROB-WL"
-    - container: "standingsearch"
-      caseType: "StandingSearch"
-      type: "jsonlines"
-      prefix: "CCD-PROB-SS"
-    - container: "divorce"
-      caseType: "DIVORCE"
-      type: "jsonlines"
-      prefix: "CCD-DIV"
+#    - container: "caveat"
```

Comment:
Commented out for testing purposes?

Reply:
I'm rolling out grantofrepresentation first and after I'll do one by one by case type to see how to avoid over-stress the  system